docs/applications/architecture.md

```diff
@@ -30,4 +30,8 @@ This diagram outlines how the three Talawa components interact
 
 ![img](/img/Talawa.jpg)
 
+### Choosing the Right Stack
```

Comment:
From users' perspective;
The description for choosing the stack should be provided before defining each one of the following. You should tell why is it important to choose the right stack.
And secondly, separate points should be made for each stack before difining and describing it.
For example,
1. Typescript
2. Flutter
3. MongoDB
Thank You!

Reply:
Thanks for the review and for your suggestions I will incorporate these changes into my request